nautobot/nautobot_data_validation_engine/datasources.py

```diff
@@ -0,0 +1,25 @@
+"""Datasource definitions."""
+
+from nautobot.extras.choices import LogLevelChoices
+from nautobot.extras.registry import DatasourceContent
+from nautobot.nautobot_data_validation_engine.custom_validators import get_classes_from_git_repo
+
+
+def refresh_git_data_compliance_rules(repository_record, job_result, delete=False):  # pylint: disable=W0613
+    """Callback for repo refresh."""
+    job_result.log("Successfully pulled git repo", level_choice=LogLevelChoices.LOG_INFO)
+    for compliance_class in get_classes_from_git_repo(repository_record):
+        job_result.log(f"Found class {compliance_class.__name__!s}", level_choice=LogLevelChoices.LOG_INFO)
```

Comment:
This feels weird/wrong to me given that the initial repo sync loads the validators into memory and logs about it, but we always reload the code on each model clean (I'm also concerned about the potential performance impact here!)